README.md

```diff
@@ -17,10 +17,21 @@ To install the package, from within Julia do
     julia> Pkg.add("ValidatedNumerics")
 
 
+## Meta-package
+
+Since version 0.9, `ValidatedNumerics.jl` is a meta-package that reexports the following packages from the [`JuliaIntervals` organization](https://github.com/JuliaIntervals) on GitHub:
+- [`IntervalArithmetic.jl`](https://github.com/JuliaIntervals/IntervalArithmetic.jl) -- fundamental arithmetic and elementary operations on intervals
+
+- [`IntervalRootFinding.jl`](https://github.com/JuliaIntervals/IntervalRootFinding.jl) -- find roots of functions in a guaranteed way
+
+- [`IntervalConstraintProgramming.jl`](https://github.com/JuliaIntervals/IntervalConstraintProgramming.jl) -- characterization of feasible sets of inequalities
+
 ## Interval arithmetic
 All calculations are carried out using **interval arithmetic**: all quantities are treated as intervals, which are propagated throughout a calculation. The final result is an interval that is *guaranteed* to contain the correct result, starting from the given initial data.
 
-The aim of the package is correctness over speed, although performance considerations are also taken into account
+The aim of the package is correctness, together with performance.
+
+
 
 ## Documentation
 Documentation is available [**here**](http://dpsanders.github.io/ValidatedNumerics.jl/).
```

Comment:
... though it currently includes all the documentation put together. Probably we should simply merge this and the update the documentation (before tagging v0.9).